packages/core/src/domain/configuration/endpointBuilder.ts

```diff
@@ -1,39 +1,24 @@
 import { BuildEnv } from '../../boot/init'
 import { timeStampNow } from '../../tools/timeUtils'
-import { generateUUID, includes } from '../../tools/utils'
+import { generateUUID } from '../../tools/utils'
 import { InitConfiguration } from './configuration'
 
 export const ENDPOINTS = {
-  alternate: {
-    logs: 'logs',
-    rum: 'rum',
-    sessionReplay: 'session-replay',
-  },
-  classic: {
-    logs: 'browser',
-    rum: 'rum',
-    // session-replay has no classic endpoint
-    sessionReplay: undefined,
-  },
-}
+  logs: 'logs',
+  rum: 'rum',
+  sessionReplay: 'session-replay',
+} as const
```

Comment:
should we start to prepare the list of breaking changes with every v4 PR? something like what we did for v3 https://github.com/DataDog/browser-sdk/blob/main/CHANGELOG.md#v300

Reply:
Sure, I'll do it in a following PR